### PR TITLE
[13.0][ADD] product_readonly_security: New module

### PR DESCRIPTION
**Pull request by victoralmau**
New module

Please @pedrobaeza and @carolinafernandez-tecnativa can you review it?

@Tecnativa TT50471

### CODE REVIEW

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM Thanks!

**Review by gbrito — APPROVED** [review]

File: product_readonly_security/models/product_mixin.py
Comment (pedrobaeza) [inline product_readonly_security/models/product_mixin.py:1]: Name the file the same as the mixin

File: product_readonly_security/models/product_mixin.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class ProductReadonlySecurityMixin(models.AbstractModel):
+    _name = "product.readonly.security.mixin"
+    _description = "Mixin to use Product Readonly Security"
+
+    @api.model
+    def check_access_rights(self, operation, raise_exception=True):
+        """Simulate that you do not have ACLs so that the create, edit and delete
+        buttons are not displayed."""
```
Comment (pedrobaeza) [inline product_readonly_security/models/product_mixin.py]: Don't overwrite the docstring, and remove the word "simulate", that makes it this as something fake, while this is totally legit. Also a bit more of explanation is needed:

```suggestion
        # Override security returning False/AccessError if not belonging to
        # the new security group. This makest that the create, edit and delete
        # buttons are not displayed.
```

File: product_readonly_security/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#. Create or edit a user and uncheck the 'Product edition' permission.
+#. Go to any product.
```
Comment (pedrobaeza) [inline product_readonly_security/readme/USAGE.rst:3]:
```suggestion
#. Login with that user.
#. Go to any product.
```

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Comment by pedrobaeza** [comment]
@sbidoul do you know what happens with this failing merge?

**Comment by sbidoul** [comment]
Could it be `setup/product_readonly_security/odoo/addons/product_readonly_security` is a regular file instead of a symlink?

**Comment by pedrobaeza** [comment]
@victoralmau did you manually create that file? Can you remove it if not to let the merge bot to regenerate it?

**Comment by pedrobaeza** [comment]
Ouch, it seems the files are needed by pre-commit. @victoralmau please run pre-commit locally again and re-push.

**Pull request merged**